tests/jsonRpc.cpp

```diff
@@ -317,6 +346,27 @@ BOOST_FIXTURE_TEST_CASE(process_arr_async, Fixture)
     BOOST_CHECK_EQUAL(jsonRpc.process(substractArray), substractResult);
 }
 
+BOOST_FIXTURE_TEST_CASE(process_arr_async_cancelled, Fixture)
+{
+    using namespace std::placeholders;
+    jsonRpc.bindAsync("subtract", std::bind(&substractArrAsyncForever, _1, _2),
+                      cancelForever);
```

Comment:
here cancelForever reads like it is going to perform a cancel operation forever

Reply:
done :)